.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```diff
@@ -0,0 +1,65 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug", "triage"]
+# assignees:
+#   - onesignal/unity
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Also tell us, what steps are required to reproduce.
+      placeholder: Tell us what's going on.
+      value: "Something isn't quite right!"
+    validations:
+      required: true
+  - type: textarea
+    id: what-are-expectations
+    attributes:
+      label: What did you expect to happen?
+      description: Also tell us, what did you expect to happen?
+      placeholder: Tell us what you see!
+      value: "A bug happened!"
+    validations:
+      required: true
+  - type: dropdown
```

Comment:
Is it possible provide an other option so they can fill it in?

Reply:
Logic is limited, so we could add an optional text box for other info. I only need to add a description to make it clear the field is to be used if other is selected.

Reply:
I went with the textbox option.